src/client/java/teammates/client/scripts/DataMigrationHelper.java

```diff
@@ -0,0 +1,34 @@
+package teammates.client.scripts;
+
+public final class DataMigrationHelper {
+
+    private static final int PRINT_CYCLE = 100;
+
+    private DataMigrationHelper() {
+        // utility class
+    }
+
+    /**
+     * Returns true if string contains no characters to be sanitized and contains substrings of sanitized characters
+     */
+    public static boolean isSanitizedHtml(String string) {
+        if (string == null) {
+            return false;
+        }
+        if (string.indexOf('<') >= 0 || string.indexOf('>') >= 0 || string.indexOf('\"') >= 0
+                || string.indexOf('/') >= 0 || string.indexOf('\'') >= 0) {
+            return false;
+        }
+        return string.indexOf("&lt;") >= 0 || string.indexOf("&gt;") >= 0 || string.indexOf("&quot;") >= 0
+               || string.indexOf("&#x2f;") >= 0 || string.indexOf("&#39;") >= 0 || string.indexOf("&amp;") >= 0;
```

Comment:
Why are these two statements written in two different ways? Is it possible to extract a utility method so that you can write this?
`return !StringHelper.containsAny("<", ">", "&alt;",  ...)`